lib/realm-verifier/src/lib.rs

```diff
@@ -7,55 +7,99 @@ use tinyvec::ArrayVec;
 
 pub const MAX_MEASUREMENT_SIZE: usize = 64;
 
-pub type MeasurementValue = ArrayVec<[u8; MAX_MEASUREMENT_SIZE]>;
+#[derive(Debug, Clone, Copy)]
```

Comment:
Remove Copy unless it's absolutely needed by some external trait bounds.

Reply:
It is required by lib/realm-verifier/src/parser/json.rs:46:17